interface-elements-for-web/articles/tree-list/filtering/creating-complex-filter-criteria-with-the-filter-control.md

```diff
@@ -0,0 +1,30 @@
+---
+title: Creating Complex Filter Criteria with the Filter Control
+author: Anna Kondratova
+---
+# Creating Complex Filter Criteria with the Filter Control
+To build complex filter criteria with an unlimited number of filter conditions, combined by logical operators, use the built-in **Filter Control**.
+
+![FilterControl](../../../images/img8592.png)
+
+Open the **Filter Control** using one of the following:
+* Click the filter image in the filter bar.
+	
+	![FilterControlShow1](../../../images/tree-list-create-filter-control.png)
+* Click the filter expression link in the filter bar.
```

Comment:
"Click the filter expression link in the filter bar" - I don't see how this relates to the corresponding image either.

Reply:
Modified.